packages/core/src/browser/performance/frontend-stopwatch.ts

```diff
@@ -0,0 +1,62 @@
+/********************************************************************************
+* Copyright (c) 2019, 2021 TypeFox, STMicroelectronics and others.
```

Comment:
Where does the 2019 date come from here?

Reply:
Much of the content of this file was factored out of another source file that was copyrighted 2019 by TypeFox and others. So it seemed prudent to maintain that copyright statement. The same applies to the backend file `node-stopwatch.ts`.